Gemfile

```diff
@@ -1,3 +1,5 @@
-source 'http://rubygems.org'
+source 'https://rubygems.org'
+
+gem 'nokogumbo', git: 'https://github.com/rubys/nokogumbo'
```

Comment:
Until Nokogumbo 2.0.0 is released.

Reply:
Let's lock this to [a `:ref` as well](https://bundler.io/v1.12/git.html) to avoid a situation where `master` might update and break things.

Reply:
Note this is the project's own Gemfile, not a runtime gem dependency, and this PR can't be merged until 2.0.0 is released, at which point the Gemfile entry will be removed. So we aren't exposing ourselves or any users to possible breakage by tracking the git branch to demonstrate the CI passes. If anything, we're avoiding surprises from subsequent prerelease updates that *would* break CI 😊